core/nixpkgs.bzl

```diff
@@ -88,10 +89,9 @@ def _nixpkgs_local_repository_impl(repository_ctx):
     else:
         target = cp(repository_ctx, repository_ctx.attr.nix_file)
 
-    repository_files = [target]
-    for dep in repository_ctx.attr.nix_file_deps:
-        dest = cp(repository_ctx, dep)
-        repository_files.append(dest)
+
+    repository_files = cp_many(repository_ctx, repository_ctx.attr.nix_file_deps)
```

Comment:
This will probably require a [similar platform check to `cp`](https://github.com/tweag/rules_nixpkgs/blob/9f08fb2322050991dead17c8d10d453650cf92b7/core/util.bzl#L40) to not break Windows users.